modules/taskbar.less

```diff
@@ -4,7 +4,7 @@
 	display: none;
 	z-index: 10001;
 	width: 50%;
-	left: 50%;
```

Comment:
Seems like I haven't given it much thought, then...
f8e45cc does indeed make more sense